plugin/evm/vm.go

```diff
@@ -79,9 +86,10 @@ const (
 
 var (
 	// Set last accepted key to be longer than the keys used to store accepted block IDs.
-	lastAcceptedKey = []byte("last_accepted_key")
-	acceptedPrefix  = []byte("snowman_accepted")
-	ethDBPrefix     = []byte("ethdb")
+	lastAcceptedKey        = []byte("last_accepted_key")
+	acceptedPrefix         = []byte("snowman_accepted")
+	ethDBPrefix            = []byte("ethdb")
+	pruneRejectedBlocksKey = []byte("pruned_rejected_blocks")
```

Comment:
We can delete this key along with `pruneChain` since it's not needed for subnet-evm.

Reply:
done